src/main/resources/migrations/00045_drop_constraint_gateway_accounts_organisation_id.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:alter_table_gateway_accounts_drop_constraint_gateway_accounts_organisation_key
+ALTER TABLE gateway_accounts DROP CONSTRAINT IF EXISTS gateway_accounts_organisation_key;
```

Comment:
Exploring in my local environment I seem to be able to update the access_token even with the unique constraint on organisation.

```
 id |               external_id               | payment_provider | service_name | type | description | analytics_id | version | access_token | organisation 
----+-----------------------------------------+------------------+--------------+------+-------------+--------------+---------+--------------+--------------
  2 | DIRECT_DEBIT:q7k1jdodep9qrnegc1bn6iifhg | SANDBOX          | My service   | TEST |             |              |       0 |              | 
  1 | DIRECT_DEBIT:3gnuulqn1vkfa2aiuf86q0vlqi | SANDBOX          | My service   | TEST |             |              |       0 | myToken      | myorg
```

Then updating the acces_token:

```
directdebit_connector=# UPDATE gateway_accounts SET access_token = 'myToke32' WHERE id=1;
UPDATE 1
directdebit_connector=# select * from gateway_accounts;
 id |               external_id               | payment_provider | service_name | type | description | analytics_id | version | access_token | organisation 
----+-----------------------------------------+------------------+--------------+------+-------------+--------------+---------+--------------+--------------
  2 | DIRECT_DEBIT:q7k1jdodep9qrnegc1bn6iifhg | SANDBOX          | My service   | TEST |             |              |       0 |              | 
  1 | DIRECT_DEBIT:3gnuulqn1vkfa2aiuf86q0vlqi | SANDBOX          | My service   | TEST |             |              |       0 | myToke32     | myorg
```

Not sure if its worth seeing checking how this works in the DAO before removing the constraint?